client/components/repeater.js

```diff
@@ -53,10 +53,7 @@ class Repeater extends Component {
   }
 
   update(items) {
-    return new Promise(resolve => {
-      this.setState({ items }, resolve)
-    })
-      .then(this.save);
+    this.setState({ items }, this.save)
```

Comment:
How is this different? The code seems the same to me.

Reply:
it was wrapped in a promise

Reply:
therefore deferred